janis_bioinformatics/tools/pmac/parsefastqc/v0_1_0.py

```diff
@@ -140,3 +148,22 @@ def bind_metadata(self):
         self.metadata.dateCreated = datetime(2020, 1, 7)
         self.metadata.dateUpdated = datetime(2020, 2, 14)
         self.metadata.version = "0.1.0"
+
+    # def tests(self):
```

Comment:
just checking, why are they commented out here?

Reply:
As we discussed before, this tool doesn't generate any output at all (at least using the data I've been given). Therefore, the test is not usable at this moment. I think I'll also add comments in the file to clarify this.

Reply:
Just curious, what happened if we leave the `output=[]`? (I don't know whether this will work or not)
If that works, we will at least force the tool to be executed without error

Reply:
Yes, that forces the tool to be executed and I get a result `Test SUCCEEDED: basic`. It seems like if I don't test the output at all (`output=[]`), the default result is Test SUCCEEDED. It can be an ad hoc solution for now.